src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -50,8 +53,16 @@ public override async Task ExecuteAsync(IContext context, ProcessHttpSettings se
                             contentType ?? "application/json");
                     }
 
+                    if (settings.Uri.AbsoluteUri.Contains(_configuration.InternalUris))
```

Comment:
move to a new method and add the rule of ";"

Reply:
ok